### PR TITLE
sample test case #3

### PR DESCRIPTION
**Pull request by rajibmp**
sample test case to get started with unit tests.

Don't forget to add PHPUnit as dev dependency in composer.json file

### CODE REVIEW

File: nep framework/tests/SampleTest.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+namespace nepframework;
```
Comment (surajprogrammez) [inline nep framework/tests/SampleTest.php]: Proper template

```
<?php
namespace Nepframework;

class SampleTest extends \PHPUnit_Framework_TestCase
{
    public function testAssertTrue()
    {
        $this->assertTrue(TRUE);
    }
}
```

**Comment by surajprogrammez** [comment]
More details for Unit testing in php using PHPUnit https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html

**Comment by surajprogrammez** [comment]
@developersanjog Also consider the template I provided in comment

File: nep framework/tests/SampleTest.php (hunk 1 above)
Comment (rajibmp) [reply nep framework/tests/SampleTest.php]: This is great, clean and more clear IMO.

Comment (developersanjog) [reply nep framework/tests/SampleTest.php]: Yes i did implement it.

Comment (developersanjog) [reply nep framework/tests/SampleTest.php]: Thank you for the guidance.

Comment (rajibmp) [reply nep framework/tests/SampleTest.php]: you are welcome. If you check other issues that I have created there are links for using different tools to add improvements in your project via Travis or Code climate, etc.

**Pull request merged**